common/common-api/src/main/java/org/ow2/proactive/scripting/TaskScript.java

```diff
@@ -67,6 +67,11 @@
      */
     public static final String CREDENTIALS_VARIABLE = "credentials";
 
+    /**
+     * The variable containing a connexion to the scheduler server
```

Comment:
connexion --> connection. It is more a proxy, the connection seems not etablished when injected.